### DISCUSSION TITLE
Replace youtube-dl with yt-dlp

### DISCUSSION THREAD

**Discussion by trimechee** [smplayer-dev/smplayer#193] [Ideas]
Hello, SMPlayer is great ! Thank you ! I propose tp Mr. Ricardo to add an option to use yt-dlp which is is a youtube-dl fork based on the now inactive youtube-dlc. The main focus of this project is adding new features and patches while also keeping up to date with the original project. 
Thank you !

https://github.com/yt-dlp/yt-dlp

**Comment by raszpl** [comment]
The  main reason to do it currently would be https://github.com/ytdl-org/youtube-dl/issues/29326 
yt-dlp implemented a workaround impersonating android client, this lets it bypass throttling (at least for now).
I just downloaded yt-dlp and renamed it to youtube-dl - works great inside smplayer/mpv/

**Reply by trimechee** [reply]
Great ! thank you for this precious information :)

**Comment by smplayer-dev** [comment]
There's support for yt-dlp in the [development version](https://github.com/smplayer-dev/smplayer/releases/tag/v21.8.0.9999), you can select it in preferences -> network,
